plugins/services/src/js/containers/pod-instances/PodInstancesTable.js

```diff
@@ -146,6 +153,20 @@ class PodInstancesTable extends React.Component {
         render: this.renderColumnAddress,
         sortable: true
       },
+      {
```

Comment:
Can we add a small test in the `PodInstancesTable-test.js` to check that these are set?

Reply:
done